### DISCUSSION TITLE
other server....

### DISCUSSION THREAD

**Discussion by ktwebktweb** [ruffle-rs/ruffle#5310] [Q&A] [answered]
hello

if I set ruffle on server A

and upload "game.swf" files to on server B

then,   server A website can play the "game.swf(server B)"  ?

I wanna uploads  swf files to server B only...

thank you so much ...!

**Comment by danielhjacobs** [comment, marked as answer]
You'll need to add an "Access Control Allow Origin" header of Server A's domain (or * to allow requests from any domain) to Server B. See https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSMissingAllowOrigin, as well as https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS for a fuller description of what CORS is. Ruffle uses the Fetch API, so is subject to the same security restrictions as other scripts.

See also #1500.

**Reply by n0samu** [reply]
I've also added [a section to our wiki](https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle#configure-cors-header) explaining it in more detail.
